README.md

```diff
@@ -386,13 +386,13 @@ await octokit.request("POST /repos/{owner}/{repo}/issues", {
 });
 ```
 
-`octokit.request` can be used to send requests to other domains by passing a full URL. It can also be used to send requests to endpoints that are not (yet) documented on [GitHub's REST API documentation](https://docs.github.com/rest).
+In both cases a given request is authenticated, retried, and throttled transparently by the `octokit` instance which also manages the `accept` and `user-agent` headers as needed.
```

Comment:
Moved this up because I'm assuming this is true for both rest and request. If not, that's a pretty big differentiator between the two and should be called out